src/libs/ReportActionsUtils.js

```diff
@@ -28,6 +28,30 @@ Onyx.connect({
     callback: val => isNetworkOffline = lodashGet(val, 'isOffline', false),
 });
 
+/**
+ * Sort an array of reportActions by their created timestamp,
+ * using actionName as a secondary sorting parameter,
+ * finally falling back on reportActionID in case multiple reportActions of the same actionName are created in the same millisecond.
+ *
+ * @param {Array} reportActions
+ * @param {Boolean} [shouldInvertSortingOrder]
+ * @returns {Array}
+ */
+function sortReportActions(reportActions, shouldInvertSortingOrder = false) {
```

Comment:
- Out of curiosity, why did you change the function name? I think `getSortedReportActions` is better and it doesn't sound like it update the report in place (by ref), like the call `sortReportActions(iouActions);` suggests
- What do you think of rename `shouldInvertSortingOrder` to be `ascending`? "should invert" doesn't say anything about the actual order so "ascending" or "descending" are better
- Can we make the `shouldInvertSortingOrder` param mandatory? There's 5 calls to this function, 3 without passing the param, and two passing true, so this isn't really a default and I think explicit params are better

Reply:
> Out of curiosity, why did you change the function name? I think getSortedReportActions is better and it doesn't sound like it update the report in place (by ref), like the call sortReportActions(iouActions); suggests 

It's because unless I'm mistaken this function technically _does_ update the report in place by ref.

Reply:
That makes sense. That said, I think we should keep the old name and always use the return value of the array instead of relying on the in-place modification (we're currently doing a mix of the two, which I don't think we should do)

Reply:
Updated the name and consistently use the return value instead of inline modification.

Reply:
> What do you think of rename shouldInvertSortingOrder to be ascending? "should invert" doesn't say anything about the actual order so "ascending" or "descending" are better

Good point about "should invert" not saying anything about the actual order, but in NewDot all boolean variables should start with `is`, `should`, `has`, or something similar. So I updated this to `shouldSortInDescendingOrder`

Reply:
> Can we make the shouldInvertSortingOrder param mandatory? There's 5 calls to this function, 3 without passing the param, and two passing true, so this isn't really a default and I think explicit params are better

I disagree about this not being a default, because the places where we pass `true` and sort in descending order are for a component called `InvertedFlatList`, where the sorting order needs to be inverted. At least that's how I reason about the default value of this parameter.